setup.py

```diff
@@ -30,6 +30,7 @@
     packages=setuptools.find_packages(),
     install_requires=[
         "marquez-python==0.7.5",
-        "sqlparse==0.3.1"
+        "sqlparse==0.3.1",
+        "google-cloud-bigquery>=1.28.0"
```

Comment:
do we actually need the google-cloud-library? It looks like not. It's better to leave off this dependency if it's not actually needed and let users have their own.

Reply:
CI failed, and i need to add those whatever CI needed.